executor/src/main/resources/logback-spring.xml

```diff
@@ -25,7 +25,13 @@
         <appender-ref ref="CONSOLE"/>
         <appender-ref ref="FILE"/>
     </logger>
-    <logger name="org.k8loud.executor.kubernetes.ResourceType" level="DEBUG" additivity="false">
+
+    <!-- TRACE can be enabled -->
+    <logger name="org.k8loud.executor.kubernetes.KubernetesResourceType" level="DEBUG" additivity="false">
```

Comment:
Trace will not work

Reply:
Intended, trace level clutters the log so I disabled it and hinted that it can be enabled